build.gradle

```diff
@@ -218,8 +218,11 @@ project(':iceberg-core') {
   dependencies {
     compile project(':iceberg-api')
     compile project(':iceberg-common')
+    compile 'com.google.guava:guava:28.0-jre'
```

Comment:
+1 on this. We experienced a problem otherwise.

Reply:
This dependency is already defined under subprojects with compileOnly scope. I think we should change the scope there to compile instead of re-adding the dependency here again since all subprojects require this dependency in a compile scope [and preferably shaded]

Reply:
Fixed for Guava and SLF4J